ccloud-monitoring/docs/consumer-use-cases.rst

```diff
@@ -0,0 +1,10 @@
+.. _ccloud-cli-tutorial-consumer-use-cases:
+
+Consumer Client Use Cases
```

Comment:
Just to standardize, would we consider renaming "Use Cases" to "Scenarios" (in all places)?  "Use case" sometimes connotes something like "What are you trying to accomplish" whereas "scenario" sometimes connotes maybe something closer to what this is?

Reply:
👍